heartbeat/docker-compose

```diff
@@ -36,7 +36,7 @@
 . ${OCF_FUNCTIONS_DIR}/ocf-shellfuncs
 
 # Defaults
-OCF_RESKEY_binpath_default=/usr/bin/docker-compose
+OCF_RESKEY_binpath_default=/usr/local/bin/docker-compose
```

Comment:
You shouldnt change the default path for it. The setting is there so you can set what the path is on your system when it's not in the default location.

Reply:
I want to change the default path aligned to Ubuntu distribution.

Reply:
You should be able to use get_release_id() for that (see ocf-distro file for logic).